fiftyone/core/stages.py

```
@@ -8136,8 +8138,18 @@
         else:
             name = None
 
-        if state != last_state or not fod.dataset_exists(name):
-            kwargs = self._config or {}
+        try:
+            last_dataset = fod.load_dataset(name)
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)